srcpkgs/ol/template

```diff
@@ -10,9 +10,26 @@ license="LGPL-3.0-or-later, MIT"
 homepage="https://yuriy-chumak.github.io/ol/"
 changelog="https://raw.githubusercontent.com/yuriy-chumak/ol/master/doc/CHANGELOG.md"
 distfiles="https://github.com/yuriy-chumak/ol/archive/${version}.tar.gz"
-checksum=1f3da2d3b2e47f5c626bfe2727b649db3fd8f53e700dad2a984314275ee7a6ff
+checksum=5a7b8ff5eeba2db63ba8d409c6ebc3af5f9cb21d14a2ec1a279d8137298360ef
 
-CFLAGS="-lm"
+case "$XBPS_TARGET_MACHINE" in
+	i686*|x86_64*) makedepends+=" gcc-multilib";;
+esac
+
+CFLAGS="${CFLAGS/-D_FORTIFY_SOURCE=2/} -lm -DOLVM_FFI -DHAS_SOCKETS -DHAS_DLOPEN" \
+
+_apply_patch() {
+	local args="$1" pname="$(basename $2)"
+
+	if [ ! -f ".${pname}_done" ]; then
+		patch -N $args -i $2
+		touch .${pname}_done
+	fi
+}
+
+post_patch() {
+	_apply_patch -p1 ${FILESDIR}/fmov.patch
+}
```

Comment:
patches should go in `srcpkgs/$pkgname/patches`, and none of this should be necessary

Reply:
I see. Done.